.gitignore

```diff
@@ -2,10 +2,6 @@
 logs
 *.log
 npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-lerna-debug.log*
-.pnpm-debug.log*
```

Comment:
We leave these because we never know when someone might be using a different package manager with this package.